src/karmabot/commands/welcome.py

```diff
@@ -41,10 +43,18 @@
 What other programming languages do you know and/or use?"""
 
 
+def _get_admins() -> str:
+    admins_env = get_env_var("KARMABOT_ADMINS", default=KARMABOT_ID).split(",")
```

Comment:
I needed to do this here in order to properly mock it in the test

Reply:
Do you need to mock it? Can't we just extend https://github.com/PyBites-Open-Source/karmabot/blob/0eaeefaa02d0e91f6125c7967b4503081234ef3f/noxfile.py#L14

Reply:
probably, as long as it has multiple admins = more realistic - how to do it please?

Reply:
In noxfile.py change it to
```
"KARMABOT_ADMINS": "FAKE_ADMIN1,FAKE_ADMIN2,FAKE_ADMIN3",
```
and then use `FAKE_ADMIN1` etc. as values in the test

Reply:
sweet, thanks, done

Reply:
so you always run your tests with `poetry run nox -rs tests`? (do we need to add `poetry run` to the readme?), I ask because this is the first test to break `poetry run pytest` if we do it this way ...

Reply:
Yes it should always be run with nox. Inside the venv you can just use the `nox` command.

If you want to run pytest directly you have to have a valid .karmabot file or env variables anyway. That was already a restriction before these changes. As some tests rely on the env variables.

Reply:
Gotcha thanks. Anything else I need to change?